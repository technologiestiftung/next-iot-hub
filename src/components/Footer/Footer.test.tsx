import { render, screen } from "@testing-library/react";
import { Footer } from ".";

describe("Footer component", () => {
  it("should render the impressum link", () => {
    render(<Footer />);
    const berlinText = screen.getByText(/Impressum/g);
    expect(berlinText).toBeInTheDocument();
  });
  it("should render the Datenschutz link", () => {
    render(<Footer />);
    const berlinText = screen.getByText(/Datenschutz/g);
    expect(berlinText).toBeInTheDocument();
  });
});
