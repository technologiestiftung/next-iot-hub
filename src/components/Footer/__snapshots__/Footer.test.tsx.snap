// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Footer Default 1`] = `
<footer>
  <section
    className="bg-white border-t border-gray-200"
  >
    <div
      className="max-w-screen-lg mx-auto p-8"
    >
      <div
        className="w-full flex"
      >
        <div
          className="block"
        >
          <span
            className="w-full block text-xs text-gray-700"
          >
            Ein Projekt der:
          </span>
          <a
            className="mt-4 block"
            href="https://technologiestiftung-berlin.de/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Logo der Technologiestiftung Berlin"
              className="w-32"
              src="/images/tsb-logo.svg"
            />
          </a>
        </div>
        <div
          className="ml-8 md:ml-12 block"
        >
          <span
            className="w-full block text-xs text-gray-700"
          >
            Gefördert von:
          </span>
          <a
            className="mt-4 block"
            href="https://www.berlin.de/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Logo der Senatsverwaltung für Wirtschaft, Energie und Betriebe"
              className="w-48"
              src="/images/senweb-logo.svg"
            />
          </a>
        </div>
      </div>
    </div>
  </section>
  <section
    className="bg-gray-100"
  >
    <div
      className="max-w-screen-lg mx-auto px-8 py-6 block sm:flex justify-between"
    >
      <div
        className="flex items-center"
      >
        <p
          className="text-sm text-gray-700"
        >
          © 
          2021
           Technologiestiftung Berlin
        </p>
      </div>
      <ul
        className="flex items-center mt-2 sm:mt-0"
      >
        <li>
          <a
            className="text-sm text-gray-700"
            href="https://www.technologiestiftung-berlin.de/de/impressum/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Impressum
          </a>
        </li>
        <li
          className="ml-4"
        >
          <a
            className="text-sm text-gray-700"
            href="https://www.technologiestiftung-berlin.de/de/datenschutz/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Datenschutzerklärung
          </a>
        </li>
      </ul>
    </div>
  </section>
</footer>
`;

exports[`Storyshots Header Default 1`] = `
<header
  className="w-full p-4 bg-white shadow-lg relative md:sticky md:top-0"
>
  <a
    className="w-max flex items-center"
    href="/"
  >
    <img
      alt="Logo des Berlin IoT Hub"
      className="w-10"
      src="/images/datahub-logo.svg"
    />
    <div
      className="ml-2"
    >
      <span
        className="font-bold"
      >
        Berlin
      </span>
       IoT Hub
    </div>
  </a>
</header>
`;
